clients/admin/src/pages/dashboard/Form.tsx

```diff
@@ -0,0 +1,169 @@
+  public state : IForm = {
+    name: "Test",
+    email: "test@test.com",
+    address: "123 Test Drive",
+    imgPath: "https://www.meatpoultry.com/ext/resources/MPImages/11-2018/11082018/wendys-exterior.jpg?1541774773",
+    phoneNum: "123-456-7890"
+  };
+
+  public handleChange = (name : keyof IForm) => (event : any) => {
+    this.setState({
+      name: event.target.value,
+    } as Pick<IForm, keyof IForm>);
+  };
+
+  public render() : JSX.Element {
+    const { classes } = this.props;
+
+    return (
+      <div>
+        <div className={classes.appBarSpacer} />
+        <Typography gutterBottom={true} component="h2">
+          Add Restaurant
+        </Typography>
+        <form className={classes.container} noValidate={true} autoComplete="off">
+          <Grid container={true} spacing={16} direction="column">
+            <Grid item={true}>
+              <TextField
+                id="name"
+                label="Name"
+                className={classes.textField}
+                value={this.state.name}
+                onChange={this.handleChange('name')}
+                margin="normal"
+              />
+              <TextField
+                id="email"
+                label="Email"
+                className={classes.textField}
+                value={this.state.email}
+                onChange={this.handleChange('email')}
+                margin="normal"
+              />
+              <TextField
+                id="address"
+                label="Address"
+                className={classes.textField}
+                value={this.state.address}
+                onChange={this.handleChange('address')}
+                margin="normal"
+              />
+            </Grid>
+            <Grid item={true}>
+              <TextField
+                id="phoneNum"
+                label="Phone Number"
+                className={classes.textField}
+                value={this.state.phoneNum}
+                onChange={this.handleChange('phoneNum')}
+                margin="normal"
+              />
+              <TextField
+                id="imgPath"
+                label="Image Path"
+                className={classes.textField}
+                value={this.state.imgPath}
+                onChange={this.handleChange('imgPath')}
+                margin="normal"
+              />
+            </Grid>
+            <Grid item={true}>
+              <Button
+                variant="contained"
+                color="primary"
+                className={classes.button}
+                onClick={
+                  () => {
+                    this.props.createItem(this.state);
+                    this.props.history.goBack();
+                  }
+                }
+              >
+                Send
+                <Send className={classes.rightIcon} />
+              </Button>
+            </Grid>
+          </Grid>
+        </form>
+      </div>
+    );
+  }
+}
+
+const styles = (theme: Theme) => ({
```

Comment:
I'm gonna nit on this now, but you can change this to:
```
const styles =(theme: Theme) => createStyles({ ... })
```
and your `IFormProps` extend ```WithStyles<typeof styles>```
so TS can infer the types from the styles